src/test/ui/editions/edition-keywords-2018-2015-parsing.stderr

```diff
@@ -38,10 +38,20 @@ error: macro expansion ends with an incomplete expression: expected one of `move
 LL |     ($i: ident) => ($i)
    |                       ^ expected one of `move`, `|`, or `||`
    | 
-  ::: $DIR/edition-keywords-2018-2015-parsing.rs:16:8
+  ::: $DIR/edition-keywords-2018-2015-parsing.rs:22:8
    |
 LL |     if passes_ident!(async) == 1 {}
    |        -------------------- in this macro invocation
 
-error: aborting due to 5 previous errors
+error[E0658]: async closures are unstable
```

Comment:
Nit: off-topic error, `#![feature(async_closure)]` can be added to the test.

Reply:
Hmm; so the purpose of this was as a "recovery witness" -- maybe I should add a more obvious instance of that. :D

Reply:
Yes, that would be better.
Also, "missing main" is also not a good recovery witness.